src/docker.rs

```diff
@@ -1091,7 +1094,7 @@ impl Docker {
             "/version",
             Builder::new().method(Method::GET),
             None::<String>,
-            Ok(Full::new(Bytes::new())),
+            Ok(body_full(Bytes::new())),
```

Comment:
Instead of using `body_full` could we just use the disambiguated type everywhere ? `BodyType::Left(Full::new(body))`

Although it's more verbose, it's a bit more explicit what's going on.

Reply:
Sure, that's fine with me. I went with the helper function mostly because it would make it easy to change the BodyType in the future, but if you prefer the disambiguated type I'll go ahead and put that in.

Reply:
Thanks for your time on this!